### PR TITLE
Tweak visibility for WPF

### PR DESCRIPTION
**Pull request by JeremyKuhne**
Added the full set of runtime assemblies with (hopefully) the right keys. Test assemblies will eventually need added.

I've introduced constants for the public key values to make this a little easier to update / follow. I didn't update all of the usages, anyone else can do so, or I'll eventually pick it off when it is convenient.

This also moves some interop down to Core and does some renaming. It is intentional that I did not change the name of some of the files to allow history to be easily kept. I'll update names to match in future pulls.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/12276)

### CODE REVIEW

File: src/System.Windows.Forms.Primitives/tests/TestUtilities/Win32/WindowClass.cs [hunk 1]
```diff
@@ -214,7 +214,7 @@ public unsafe HWND CreateWindow(
         }
     }
 
-    protected virtual LRESULT WNDPROC(HWND hWnd, MessageId msg, WPARAM wParam, LPARAM lParam)
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms.Primitives/tests/TestUtilities/Win32/WindowClass.cs:217]: Why did you have to change MessageId to a uint? Message Id seems to be a reasonable class to share.

File: src/System.Private.Windows.Core/src/PublicKeys.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    /// </summary>
+    public const string MicrosoftShared = "0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9";
+
+    /// <summary>
+    ///  The standard Microsoft ECMA key. The token is b77a5c561934e089.
+    /// </summary>
+    public const string Ecma = "00000000000000000400000000000000";
+
+    /// <summary>
+    ///  The open key. The token is cc7b13ffcd2ddd51.
```
Comment (Tanya-Solyanik) [inline src/System.Private.Windows.Core/src/PublicKeys.cs:19]: Do you mean - "The key used with open source assemblies."

**Review by Tanya-Solyanik — APPROVED** [review]

File: src/System.Private.Windows.Core/src/PublicKeys.cs (hunk 2 above)
Comment (JeremyKuhne) [reply src/System.Private.Windows.Core/src/PublicKeys.cs:19]: I’m just copying the details I have at the moment

File: src/System.Windows.Forms.Primitives/tests/TestUtilities/Win32/WindowClass.cs (hunk 1 above)
Comment (JeremyKuhne) [reply src/System.Windows.Forms.Primitives/tests/TestUtilities/Win32/WindowClass.cs:217]: Maybe we’ll share it eventually

**Pull request merged**